tasks/section1.yml

```diff
@@ -1292,7 +1302,12 @@ print $NF }'"
       - rule_1.7.1.6
 
 - name: "NOTSCORED | 1.7.1.6 | PATCH | Ensure permissions on /etc/issue.net are configured"
-  command: /bin/true
+  file:
+      dest=/etc/issue.net
+      state=file
+      owner=root
+      group=root
+      mode=0644
```

Comment:
This task needs to be in YAML literal form per https://github.com/ansible/ansible-lockdown/blob/master/CONTRIBUTING.md